internal/dataplane/parser/translate_udproute.go

```diff
@@ -53,14 +53,22 @@ func (p *Parser) ingressRulesFromUDPRoutes() ingressRules {
 }
 
 func (p *Parser) ingressRulesFromUDPRoute(result *ingressRules, udproute *gatewayapi.UDPRoute) error {
-	// first we grab the spec and gather some metadata about the object
 	spec := udproute.Spec
 
-	// each rule may represent a different set of backend services that will be accepting
+	// Validation for TCPRoutes will happen at a higher layer, but in spite of that we run
```

Comment:
Might mention what is the "higher level" here

Reply:
Thanks for pointing out. I gave it a second thought and this comment is misleading because there is no additional validation that out-of-the-box GWAPI CRD validation for `TCPRoute` and `UDPROute`